### ISSUE TITLE
MultiSelect firing valueChange for every value in a summary tag on summery tag removal

### ISSUE THREAD

**Issue by jiangyh1024** [telerik/kendo-angular#2457]
![image](https://user-images.githubusercontent.com/34930870/63997439-12c13a00-cb31-11e9-8de6-405c0fcd725c.png)

after clicking 'x', valueChange is triggered (24 + 1) times.

I think it should  only be triggered one time

```
 <kendo-multiselect #schedulerSelect [popupSettings]="{popupClass: 'combobox-with-check-all'}"
              (valueChange)="onDetailDataChanged()" formControlName="userCommodities" textField="text"
              valueField="value" [data]="dsComm" [autoClose]="false" [valuePrimitive]="true"
              [kendoMultiSelectSummaryTag]="3">
              <ng-template kendoMultiSelectGroupTagTemplate let-dataItems>
                <div kendoTooltip *ngIf="dataItems.length > 0">
                  <span [title]="getTitle(dataItems)">{{ dataItems.length }} more selected</span>
                </div>
              </ng-template>

              <ng-template kendoMultiSelectHeaderTemplate let-dataItem>
                <div class="check-all-container">
                  <input (change)="onCheckAllClicked($event, 'userCommodities')" [(ngModel)]="isSchedulerCommCheckAll"
                    [ngModelOptions]="{standalone:true}" id="userCommodities-check-all" class="k-checkbox"
                    type="checkbox" />
                  <label style="width: 100%;" for="userCommodities-check-all" class="k-checkbox-label">
                    Check All</label>
                </div>
              </ng-template>

              <ng-template kendoMultiSelectItemTemplate let-dataItem>
                <div>
                  <input class="k-checkbox" type="checkbox"
                    [checked]="isItemSelected(dataItem.value, 'userCommodities')" />
                  <label class="k-checkbox-label">
                    {{dataItem.text}}</label>
                </div>
              </ng-template>

            </kendo-multiselect>
```

**Comment by jiangyh1024** [comment]
![image](https://user-images.githubusercontent.com/34930870/63997980-9deeff80-cb32-11e9-8865-805b0df7b911.png)

unselect will be triggered 24 times thus causing valuechange to be triggered 24 times two

![image](https://user-images.githubusercontent.com/34930870/63997962-929bd400-cb32-11e9-8296-aefdbcd642f6.png)

and there is another extra valueChange that will be triggered after above operations

![image](https://user-images.githubusercontent.com/34930870/63998026-c1b24580-cb32-11e9-8588-95f9e3e798ea.png)

**Comment by jiangyh1024** [comment]
![image](https://user-images.githubusercontent.com/34930870/63998136-0342f080-cb33-11e9-8aa9-a3e5fe30568b.png)
![image](https://user-images.githubusercontent.com/34930870/63998146-08a03b00-cb33-11e9-890f-2650e903d3d7.png)
![image](https://user-images.githubusercontent.com/34930870/63998163-11910c80-cb33-11e9-81cd-db4b3cd90486.png)

![image](https://user-images.githubusercontent.com/34930870/63998173-15249380-cb33-11e9-89ce-7d73cee8496a.png)

**Comment by jiangyh1024** [comment]
Hope that this can be fixed as soon as possible

**Comment by siyka-marinova** [comment]
@jiangyh1024 Fix for this issue is now available in the [latest dev. version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/#toc-obtaining-development-builds) of `@progress/kendo-angular-dropdowns` package

**Comment by dimitar-pechev** [comment]
Released in **v4.2.4**.
